src/framework/preferences/js/SelfVoicingEnactor.js

```diff
@@ -0,0 +1,151 @@
+/*
+Copyright 2013-2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+var fluid_2_0 = fluid_2_0 || {};
+
+(function ($, fluid) {
+    "use strict";
+
+    /*******************************************************************************
+     * speak
+     *
+     * An enactor that is capable of speaking text.
+     * Typically this will be used as a base grade to an enactor that supplies
+     * the text to be spoken.
+     *******************************************************************************/
+
+    fluid.defaults("fluid.prefs.enactor.speakEnactor", {
+        gradeNames: ["fluid.prefs.enactor", "autoInit"],
+        preferenceMap: {
+            "fluid.prefs.speak": {
+                "model.value": "default"
+            }
+        },
+        model: {
+            value: false
+        },
+        events: {
+            onStart: null,
```

Comment:
Do you have an ETA for when that will go in. We will need the TTS stuff for our First Discovery Tool work. The "deliverable" of the First Discovery Tool is due Feb 13.